inference-engine/src/plugin_api/ie_ngraph_utils.hpp

```diff
@@ -153,3 +155,97 @@ INFERENCE_ENGINE_API_CPP(CNNNetwork) cloneNetwork(const CNNNetwork& network);
 
 }  // namespace details
 }  // namespace InferenceEngine
+
+namespace ov {
+namespace runtime {
+inline ::ov::element::Type convertPrecision(const ie::Precision& precision) {
+    ie::Precision::ePrecision pType = precision;
```

Comment:
why `inline ::ngraph::element::Type convertPrecision(const Precision& precision)` does not work for us?

Reply:
replaced